lib/activity_model.dart

```diff
@@ -22,11 +22,11 @@ class ActivityModel extends Model {
   List<Map<String, dynamic>> _redoStack = [];
   Function _saveCallback;
   Popped _popped = Popped.noPopup;
-  String _highlighted;
+  String _highlighted = 'assets/menu/svg/pencil';
   String _imagePath;
 
-  bool _isDrawing = false;
-  bool _isdotSketch = false;
+  bool _isDrawing = true;
```

Comment:
Is this OK? previously it was initialized to false and now it is true

Reply:
bool _isDrawing = true, because at the first time on canvas pencil is enabled now.